strings/salesman_replacement.json

```diff
@@ -0,0 +1,49 @@
+{
+
+    "salesman": {
+        "shuttle": "cungadero",
+        "clam": "pipis",
+        "clams": "pipis",
+        "egg": "pipis",
+        "eggs": "pipis",
+        "love": "hyperlink blocked",
+        "captain": "big shot",
+        "dollar": "kromer",
+        "dollars": "kromer",
+        "credit": "kromer",
+        "credits": "kromer",
+        "basement": "deep abyss",
+        "mech": "workout-ready body",
+        "exosuit": "workout-ready body",
+        "phazon": "NEO",
+        "bolts": "high quality encryption",
+        "jester": "clown around town",
+        "salesman": "number 1 rated saleman1997",
+        "soul": "HeartShapedObject",
+        "souls": "hearts",
+        "bootstraps": "silly strings",
+        "mediocre": "$4.99",
+        "boring": "$4.99",
+        "cash": "wacky stacks",
+        "dump": "home-made storefront site",
+        "dumpster": "home-made storefront site",
```

Comment:
not only is this going to never be said in common terminology, it will actually be accidentally replaced when people actually trying to roleplay as the character say stuff. in the [game of which must not be mentioned] the man says dumpster itself, so... you see where i'm going with this

Reply:
actually he doesnt say dumpster, he says garbage can. Dumpster is used to refer to his home-made storefront site.